src/app/clients/ExpandedClientDetailsFallback.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Skeleton } from "@mui/material";
+import styled from "styled-components";
+
+const SkeletonKey = styled(Skeleton)`
+    display: inline-block;
+    border-radius: 0.7em;
+    padding: 0.2em 0.5em;
+`;
+
+const SkeletonValue = styled(Skeleton)`
+    padding: 0.2em 0.5em;
+`;
+
+const SkeletonItem = styled.div`
+    padding-bottom: 1em;
+`;
+
+const SkeletonDiv = styled.div`
+    width: 1000px;
+    max-width: 100%;
+`;
+
+const clientDetailFields = [
+    "VOUCHER #",
+    "FULL NAME",
+    "PHONE NUMBER",
+    "PACKING DATE",
+    "PACKING TIME",
+    "DELIVERY_INSTRUCTIONS",
+    "ADDRESS",
+    "HOUSEHOLD",
+    "AGE & GENDER OF CHILDREN",
+    "DIETARY REQUIREMENTS",
+    "FEMININE PRODUCTS",
+    "BABY PRODUCTS",
+    "PET FOOD",
+    "OTHER REQUIREMENTS",
+    "EXTRA INFORMATION",
+];
+
+const ExpandedClientDetailsFallback: React.FC<{}> = () => {
+    return (
+        <SkeletonDiv>
+            {clientDetailFields.map((field, index) => {
+                return (
+                    <SkeletonItem key={index}>
+                        <SkeletonKey
+                            variant="text"
+                            width={field.length * 10}
```

Comment:
That's a good spot. I've sorted it now so that really long headers now wrap instead of overflowing. The height is slightly off in some cases, but it's close enough not to be too much of a problem

Reply:
I noticed it is working when the long heading has space, but it is still not working when I put e.g. "large_field______________________________________________________________" (For the Modal I used `
    overflow-wrap: break-word` to fix it, not sure if your skeleton page allows this, although this is more like an edge case)